tests/test_excelparser/result_ontology/fromexcelonto.ttl

```diff
@@ -12,7 +12,7 @@
         "Jesper Friis"@en,
         "Sylvain Gouttebroze"@en ;
     dcterms:title "A test domain ontology"@en ;
-    owl:imports <http://emmo.info/emmo-inferred>,
+    owl:imports <http://emmo.info/emmo>,
```

Comment:
I do not understand why. We import beta-4 which should not have changed as development is now happening in beta5.

Reply:
It is due to backward corrections of ontology IRIs on GitHub Pages. Probably a bad decision, but now it is done and should not be reverted again...